File: app/src/main/java/com/hhy/game/snowman/game/VectorXY.java
package com.hhy.game.snowman.game;

public class VectorXY {
    private int x;
    private int y;

    VectorXY(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public int getX() {
        return x;
    }

    int getY() {
        return y;
    }

    void set(int x, int y) {
        this.x = x;
        this.y = y;
    }

    void set(VectorXY src) {
        this.x = src.getX();
        this.y = src.getY();
    }
}

File: app/src/main/java/com/hhy/game/snowman/game/VectorXY.kt
package com.hhy.game.snowman.game

class VectorXY internal constructor(var x: Int, var y: Int) {

    operator fun set(x: Int, y: Int) {
        this.x = x
        this.y = y
    }

    fun set(src: VectorXY) {
        x = src.x
        y = src.y
    }

}